jsonlinter.py

```diff
@@ -0,0 +1,98 @@
+# python3 lintjson.py
```

Comment:
Is this a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix))? If so, it should be like this:
```suggestion
#! python3
```
Even if not, it seems the file name has changed.
The shebang allows unix users to run `./jsonlinter.py` as an executable (with proper permissions) and the shell will know to instead run `python3 jsonlinter.py`. I'm a big fan.

Reply:
I actually intended that to be the title but running the script with a shebang is a good idea.